### PR TITLE
feat(css_parser): allow multiple semicolons after declaration

### PR DESCRIPTION
**Pull request by fireairforce**
## Summary

closes: #3836 

I introduce a new syntax for css parser which named `CSS_EMPTY_DECLARATION`, you can see it at my change at `css.ungram`, the `CSS_EMPTY_DECLARATION` just use to deal the multiple `;`  at `CSS_DECLARATION_LIST` which like JS parser's `JS_EMPTY_STATEMENTS`:

```js
let a = 1;;;;;
```

image

## Test Plan

I add test case for `biome_css_parser` like:

```css
.a {
  color: red;;;;
  color: blue;
}
```

And update some test case of `biome_css_formatter`.

### CODE REVIEW

**Comment by fireairforce** [comment]
this mr is ready for review~

**Comment by fireairforce** [comment]
ok, i will update the template later.  i try to update the `css.ungram`: 

```bash
CssDeclarationWithSemicolon =
    declaration: CssDeclaration
    ';'*
```

But it will get an panic like: 

image

And I've looked elsewhere and haven't seen similar writing methods. So i refer the statement `JS_EMPTY_STAMENT` of js parser.

File: crates/biome_css_parser/src/syntax/mod.rs [hunk 1]
```diff
@@ -232,11 +232,27 @@ pub(crate) fn parse_declaration_with_semicolon(p: &mut CssParser) -> ParsedSynta
...
+    }
+}
+
 #[inline]
 fn is_at_declaration_important(p: &mut CssParser) -> bool {
     p.at(T![!]) && p.nth_at(1, T![important])
 }
 
+#[inline]
+pub(crate) fn is_at_declaration_semion(p: &mut CssParser) -> bool {
```
Comment (ematipico) [inline crates/biome_css_parser/src/syntax/mod.rs:252]: Is this a typo? What does "semion" mean?

File: crates/biome_css_formatter/src/css/auxiliary/empty_declaration.rs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+use crate::prelude::*;
+use biome_css_syntax::CssEmptyDeclaration;
+use biome_rowan::AstNode;
+#[derive(Debug, Clone, Default)]
+pub(crate) struct FormatCssEmptyDeclaration;
+impl FormatNodeRule<CssEmptyDeclaration> for FormatCssEmptyDeclaration {
+    fn fmt_fields(&self, node: &CssEmptyDeclaration, f: &mut CssFormatter) -> FormatResult<()> {
+        format_verbatim_node(node.syntax()).fmt(f)
```
Comment (ematipico) [inline crates/biome_css_formatter/src/css/auxiliary/empty_declaration.rs:8]: This must be implemented

**Review by ematipico — COMMENTED** [review]
One thing missing, then we can merge it

File: crates/biome_css_formatter/src/css/auxiliary/empty_declaration.rs (hunk 2 above)
Comment (fireairforce) [reply crates/biome_css_formatter/src/css/auxiliary/empty_declaration.rs]: sorry, i forget here, i implement this method and add a test case for this at `declaration.css` test file

**Review by ematipico — APPROVED** [review]
Amazing work @fireairforce, thank you very much for this contribution! ❤️

**Pull request merged**